modules/claims_api/lib/bgs_service/vet_record_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VetRecordService < ClaimsApi::LocalBGS
+    def bean_name
+      'VetRecordServiceBean/VetRecordWebService'
+    end
+
+    def update_birls_record(**options)
+      poa_code = options[:poa_code]
+      body = Nokogiri::XML::DocumentFragment.parse <<~EOML
+        <birlsUpdateInput>
+          <CLAIM_NUMBER>#{options[:file_number]}</CLAIM_NUMBER>
+          <SOC_SEC_NUM>#{options[:ssn]}</SOC_SEC_NUM>
+          <POWER_OF_ATTY_CODE1>#{poa_code[0]}</POWER_OF_ATTY_CODE1>
+          <POWER_OF_ATTY_CODE2>#{poa_code[1]}#{poa_code[2]}</POWER_OF_ATTY_CODE2>
```

Comment:
Just double-checking that we want the POA code broken up, i.e., poa_code = '083' becomes:

```xml
<POWER_OF_ATTY_CODE1>0</POWER_OF_ATTY_CODE1>
<POWER_OF_ATTY_CODE2>83</POWER_OF_ATTY_CODE2>
```

Reply:
@tycol7 I thought the same thing, @FonzMP pointed out [this line](https://github.com/department-of-veterans-affairs/bgs-ext/blob/master/lib/bgs/services/vet_record.rb#L35) in bgs-ext.